automerge-backend-wasm/scripts/js_tests.sh

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+
+if [[ -e "../../automerge" ]]
+then
+    echo "Automerge repo exists. Skipping cloning..."
```

Comment:
Is this likely to lead to issues where one has checked out the code but then don't update it when we change the `automerge` commit we depend on leading to tests that fail on a fresh repo but not on the out of date checkout? If so we should check the specific commit version in this script

Reply:
I imagine the following usage:
- Locally, you probably have an `automerge` folder in the same directory as `automerge-rs`. When developing, you don't want to blow away your local copy of `automerge` with the commit version specified in this script.
- In CI, you probably want to test with a specific commit version and there won't be a `automerge` folder in the same directory.

Reply:
Ah interesting, I was assuming this would be a checkout specifically for testing. I can see how that might be difficult if you're developing both a JS and Rust change at once (which is not uncommon). 

I wonder if an approach like providing an environment variable to override the JS location would work. If the variable is set we skip the commit version check and just use whatever we find there.

That means the default case will be the same as CI so hopefully avoiding some of the more annoying reproducibility accidents that can happen because you have to explicitly set an env bar, which hopefully is visually obvious